demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1119,6 +1120,11 @@ def add_to_list(self, object_type: IDSetType, obj):
 
         self._id_set_dict.setdefault(object_type, []).append(obj) if obj not in self._id_set_dict[object_type] else None
 
+    def add_to_dict(self, object_type: IDSetType, obj_name, obj_value):
+        if not IDSetType.has_value(object_type):
+            raise ValueError(f'Invalid IDSetType {object_type}')
```

Comment:
Let's make this a more readable and understandable error

Reply:
remove this error message, it always needs to be "Packs"